src/main/kotlin/com/github/mdashl/rankedhelper/command/WdrCommand.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.mdashl.rankedhelper.command
+
+import com.github.mdashl.rankedhelper.commandhandler.Command
+import com.github.mdashl.rankedhelper.playerSP
+import com.github.mdashl.rankedhelper.utility.sendMessage
+
+object WdrCommand : Command("wdr") {
+
+    override fun execute(args: List<String>) {
+        if (args.isEmpty()) {
+            sendMessage("§cPlease, provide a player to report!")
+            return
+        }
+
+        val player = args.first()
+
+        playerSP.sendChatMessage("/wdr $player ka antikb reach speed fly autoclicker")
```

Comment:
Rename playerSP to thePlayer.
```suggestion
        thePlayer.sendChatMessage("/wdr $player ka antikb reach speed fly autoclicker")
```